lib/commands/create.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
 
 function create (opts) {
   opts.resume = false // cannot resume for create
-  if (opts._.length) opts.dir = opts._[0]
+  if (opts._.length && opts.dir === process.cwd()) opts.dir = opts._[0] // use first arg as dir if default set
```

Comment:
does this pattern also work for comands that contain home dirs like `dat sync ~/path/to/mydata`